manifests/init.pp

```diff
@@ -977,7 +982,15 @@
   if $puppet_manage_backups == true {
     notice('Puppet will manage backups because $puppet_manage_backups is true')
     include ::gitlab::install
-    include ::gitlab::backup
```

Comment:
You are proposing the use of effectively global variables rather than passing them neatly to classes?